src/Parser/BloodDeathKnight/Modules/Items/T20_2pc.js

```diff
@@ -11,22 +11,37 @@ import Combatants from 'Parser/Core/Modules/Combatants';
 class T20_2pc extends Module {
   static dependencies = {
     combatants: Combatants,
-  }
+  };
 
+  uptime=0;
```

Comment:
This could be a getter;

```jsx
get uptime() {
  return this.combatants.getBuffUptime(SPELLS.GRAVEWARDEN.id) / this.owner.fightDuration;
}
```